src/main/java/com/github/tomakehurst/wiremock/extension/ClientExtensions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2024 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.extension;
+
+import static java.util.stream.Collectors.toMap;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+public class ClientExtensions extends Extensions {
+
+  public ClientExtensions(ExtensionDeclarations extensionDeclarations) {
```

Comment:
I believe `ClientExtensions` should accept a boolean to indicate `isExtensionScanningEnabled`, so that it can `loadExtensionsAsServices` similarly to `ServerExtensions`.